app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal():
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> str:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
The `feed` method of the `Animal` class can be simplified. You don't need to use an `else` statement here. If the condition in the `if` statement is not met, the function will automatically continue to the next line after the `if` block. This is related to the first point in the 'Make Code Easier' section of the checklist.